packages/w3/bin.js

```diff
@@ -13,12 +13,13 @@ cli
 cli.command('token')
   .option('--api', 'URL for the Web3 Storage API. Default: https://api.web3.storage')
   .option('--delete', 'Delete your saved token')
-  .describe('Save an API token to use for all requests.')
+  .describe('Save an API token to use for all requests')
   .action(token)
 
 cli.command('put <path>')
   .describe('Upload a file or directory to web3.storage')
-  .option('--no-wrap', 'Dont wrap input files with a directory.')
+  .option('--no-wrap', 'Dont wrap input files with a directory')
+  .option('--no-retry', 'Dont try the upload again if it fails')
```

Comment:
```suggestion
  .option('--no-retry', 'Dont try the upload again if it fails')
  .option('--retry', 'Set maximum number of retries for client put. Default: 5')
```

Reply:
Ah, that was intentional. I decided not to include the "try n times" option explicitly in the help as I think folks should either stick with the default, or turn it off if they are debugging.